docs/main/user-guides/upgrade.md

```diff
@@ -20,57 +20,139 @@ Upgrade to the latest cStor or Jiva version is supported only from 1.12.0 or lat
-  ```
-  The `CHILD-STATUS` column of the output from the above command should not have the value as `Degraded`.
-
-2. Once the above requirements are met, execute the `helm upgrade` command to upgrade the openebs/openebs helm chart to version 3.6.0.
+Upgrade for Mayastor is supported from v2.0.x to v2.1.0 and above. Before we get into the upgrade documentation for Mayastor, it is important that we understand that there are two ways of deploying Mayastor (since v2.0.0) on to a kubernetes cluster -- the ['mayastor' helm chart](https://github.com/openebs/mayastor-extensions/tree/develop/chart) and the ['openebs' helm chart](https://github.com/openebs/charts/tree/main/charts/openebs). The upgrade instructions differ for these two deployment strategies. The first step of the upgrade is to identify the helm chart that you have used to deploy Mayastor. If you are using the Mayastor with the openebs helm chart, then the instructions for upgrading the mayastor helm chart do not apply to you.
 
-:::note
-The installation parameters present in [values.yaml](https://github.com/openebs/charts/blob/main/charts/openebs/values.yaml) can be configured based on the requirements. 
-:::
+### Indentifying the helm chart
 
-3. Next, to upgrade the data plane component, select a node that has an `io-engine` pod deployed on it. To get the list of such nodes, execute
+You'll need the helm v3 binary to be on you $PATH environment variable. Execute the following command to check if you have helm v3:
 ```
-kubectl get node -l openebs.io/engine=mayastor
+$ helm version --short
+v3.9.4+gdbc6d8e
 ```
 
-From the list of nodes obtained using the above command, select a node and perform the Mayastor node-drain operation on it.
+Execute the following command to list the helm releases on the namespace where Mayastor is installed:
 ```
-kubectl mayastor drain node <node-name> <label>
+$ helm list -n openebs
+NAME            	NAMESPACE	REVISION	UPDATED                             	STATUS  	CHART        	APP VERSION
+openebs-mayastor	openebs  	1       	2023-05-26 22:03:15.473035 +0530 IST	deployed	openebs-3.6.0	3.6.0   
 ```
-:::note
-Add a custom drain-label to the above drain process in order to ensure no conflicts with any other node-drain/node-cordon labels. 
-:::
+If the value of the 'CHART' column in the row which contains the helm chart for Mayastor, in the output of the above command, reads 'openebs-...', then your Mayastor deployment uses the openebs helm chart. If the CHART column instead reads 'mayastor-...', then it makes use of the mayastor helm chart.
+[Click here if your Mayastor deployment uses the 'mayastor' helm chart](https://mayastor.gitbook.io/introduction/additional-information/upgrade).
+
+Continue down this page if your Mayastor deployment uses the 'openebs' helm chart.
+
+### Upgrading Mayastor installed from the openebs helm chart
+
+#### 1. Planning the upgrade
+Supported upgrade paths for upgrade for Mayastor, installed with the openebs chart, are...
```

Comment:
of Mayastor

Reply:
Going with "Supported upgrade paths for upgrade for Mayastor..."